src/app/modules/shared-components/components/path-suggestion/path-suggestion.component.html

```diff
@@ -19,7 +21,7 @@
   <ng-container *ngIf="state.isReady">
     <ng-container *ngIf="state.data; else noPath">
       <ng-container *ngIf="state.data?.length === 1 && state.data?.[0]?.length === 1; else pathsSection">
-        <alg-error class="dark small" *ngIf="state.data?.[0]?.[0] as rootItem">
+        <alg-error class="alg-flex-1 light" *ngIf="state.data?.[0]?.[0] as rootItem">
           <span i18n>
             This content is one of your root
```

Comment:
Maybe this message should be shown with the same font size as "This content can be found in..." 
![Screenshot 2023-07-17 at 16 12 43](https://github.com/France-ioi/AlgoreaFrontend/assets/1053150/a706c290-3da6-4cb1-b853-722160328970)
![Screenshot 2023-07-17 at 16 13 12](https://github.com/France-ioi/AlgoreaFrontend/assets/1053150/fb3a3152-8c83-4832-82f7-1cd3fe93aa97)

Reply:
Fixed